core/src/test/java/org/infinispan/distribution/DistSyncFuncTest.java

```diff
@@ -39,6 +42,32 @@ public DistSyncFuncTest() {
       testRetVals = true;
    }
 
+   protected Map<Cache<?, ?>, ReplListener> listenerLookup;
```

Comment:
-1
I would prefer to have an empty `waitForInvalidateL1InNonOwners()` method here and override it in the async test instead of moving "unused" listeners to the sync test.